### PR TITLE
fix for issue-1711-: Metrics not getting update correctly for eventlistener_event_count

### PR DESCRIPTION
**Pull request by sudhishmk**
fix for issue-1711-: Metrics not getting update correctly for eventlistener_event_count

[Issue](https://github.com/tektoncd/triggers/issues/1711) -:  eventlistener_event_count metrics is not updated correctly
~~~
eventlistener_client_results{name=""} 34
# HELP eventlistener_event_count Number of events sent
# TYPE eventlistener_event_count counter
eventlistener_event_count{event_source="",event_type="",name="",namespace_name="",resource_group="",response_code="",response_code_class="",response_error="",response_timeout=""} 404
# HELP eventlistener_go_alloc The number of bytes of allocated heap objects
~~~

Metrics Output After the changes
~~~
# HELP eventlistener_event_received_count number of events received by sink
# TYPE eventlistener_event_received_count counter
eventlistener_event_received_count{status="succeeded"} 1
~~~

Fix -: Metric name updated from " eventlistener_event_count" to "eventlistener_event_received_count". 
Docs updated

# Release Notes

```release-note
NONE
```

### CODE REVIEW

File: docs/eventlisteners.md [hunk 1]
```diff
@@ -620,7 +620,7 @@ The following pipeline metrics are available on the `eventlistener` Service on p
 |  Name | Type | Labels/Tags | Status |
 | ---------- | ----------- | :-: | ----------- |
 | `eventlistener_triggered_resources` | Counter | `kind`=&lt;kind&gt; | experimental |
-| `eventlistener_event_count` | Counter | `status`=&lt;status&gt; | experimental |
+| `eventlistener_event_received_count` | Counter | `status`=&lt;status&gt; | experimental |
```
Comment (savitaashture) [inline docs/eventlisteners.md:623]: Hi @sudhishmk :wave: 
I m not able to understand here how the metrics issue getting solved by changing the name

Comment (sudhishmk) [reply docs/eventlisteners.md:623]: @savitaashture  Issue seems to be caused by conflict with this [metric](https://github.com/tektoncd/triggers/blob/9b472899801f88cd700fe3b99bf6bb7fc88af494/vendor/knative.dev/eventing/pkg/metrics/source/stats_reporter.go#L32-L36) with same name.

**Review by khrm — COMMENTED** [review]
@sudhishmk Can you run gofmt on the files? CI tests are failing due to this.

File: docs/eventlisteners.md (hunk 1 above)
Comment (khrm) [reply docs/eventlisteners.md:623]: `knative/pkg` has added a new metric with the same name. So we need to change our metrics to some other name.

**Comment by sudhishmk** [comment]
> @sudhishmk Can you run gofmt on the files? CI tests are failing due to this.

Done

**Review by khrm — COMMENTED** [review]
/approve

File: docs/eventlisteners.md (hunk 1 above)
Comment (savitaashture) [reply docs/eventlisteners.md:623]: ah ok got it :+1:

**Pull request merged**